src/lines.jl

```diff
@@ -1,5 +1,12 @@
 # Line plots
-import .LibCImPlot: PlotLine
+
+function PlotLine(label_id, x::AbstractArray{T}, y::AbstractArray{T}, count::Integer, offset::Integer = 0, stride::Integer = sizeof(T)) where {T<:ImPlotData}
+    LibCImPlot.PlotLine(label_id, x, y, count, offset, stride)
+end
+
+function PlotLine(label_id, x::AbstractArray{T}, y::AbstractArray{T}, count::Integer, offset::Integer = 0, stride::Integer = sizeof(Float64)) where {T<:Real}
+    LibCImPlot.PlotLine(label_id, Float64.(x), Float64.(y), count, offset, stride)
+end
```

Comment:
@sairus7 like this example to add method signatures so we can do both kwargs and implot C++ style arguments

Reply:
More or less done now for all the plot types we currently have wrapped.

Reply:
Just out of curiosity, are these codes auto-generated?

Reply:
No. But I'm thinking if we parse the metadata in [cimplot's definitions.json output](https://github.com/cimgui/cimplot/blob/47c2f24075910cfd9ddbf55b1815607dbade2e8d/generator/output/definitions.json#L8949-L8953) and do something like what I started [here](https://github.com/wsphillips/ImPlot.jl/blob/70df75dde3a31a638d78876732329b8bf1fbf53e/gen/generator.jl#L82-L86) we could indeed automate defaults for the whole thing...